zerver/tests/test_signup.py

```diff
@@ -608,4 +608,50 @@ def test_create_realm(self):
             self.assertTrue(result["Location"].endswith("/invite/"))
 
             result = self.client.get(result["Location"])
-            self.assert_in_response("You're the first one here!", result)
+            self.assertIn("You're the first one here!", result.content)
```

Comment:
Why this change?

This is incorrect because `result.content` is a bytestring. Comparing a string to a bytestring in python 3 gives `TypeError: 'str' does not support the buffer interface`.